### PR TITLE
make check: fail on any error

### PR DESCRIPTION
**Pull request by jelmd**
Scripts used for checking now propagate errors so that `make check` now fails as well.

Per default `make check` (and related scripts) stop immediately on any error for easier troubleshooting. To get all errors in a row (i.e. do not stop after an error) run `FAIL_EARLY=0 make check`.

TBD/See also: #2044

### CODE REVIEW

**Comment by jelmd** [comment]
The problem is now fixed. However, before merging it into the main branch the tests should be fixed for obvious reasons ;-)

**Comment by b-gehrke** [comment]
I'd like to work on the failing tests but as you are merging from your forked repository I cannot push to this branch. Can you please push your branch to the Hets repository?

**Comment by jelmd** [comment]
> ... I cannot push to this branch. 

Hmmm, strange. As a member of this org it should be possible ....

> Can you please push your branch to the Hets repository?

Yepp, no poblem: done.

**Comment by b-gehrke** [comment]
> Hmmm, strange. As a member of this org it should be possible ....

It worked now. Probably some git remote stuff that I messed up.

Most OWL2 tests success now. There are 2 failing. The are OWL2/tests/{4.dol,5.dol}. The problem appears to be such a renaming:
```
with Class Woman |-> Female, NamedIndividual Peter |-> John
```

The current datatype for SymbMap Items. Allows only to specify one EntityType followed by multiple IRI pairs
```haskell
data SymbMapItems = SymbMapItems ExtEntityType [(IRI, Maybe IRI)]
```

Therefore, the code above is parsed as follows
```haskell
SymbMapItems (EntityType Class) [(A,Just B),(NamedIndividual,Nothing)]
```
@tillmo, @mcodescu It seems like `Syntax.Parse_AS_Structured.parseMapOrHide` should do a second call to `OWL2.symbItemsMap`    (Might be related to  64bbf05?). Maybe we restrict the symbol IRIs not to be a curie with omitted empty prefix and an entity type keyword? For example, if one wants to rename a class `:NamedIndividual`, one would write `with Class :NamedIndividual |-> OtherIndividual`. Writing `with Class NamedIndividual |-> OtherIndividual` would then be forbidden.

**Comment by tillmo** [comment]
yes, the problem is that we currently do not allow the `EntityType` be specified for each symbol pair. This should be changed. See function `symbMapItems` in `CASL.SymbolParser`.

File: OWL2/tests/test10.dol [hunk 1]
```diff
@@ -1,4 +1,4 @@
-logic OWL
+logic OWL serialization Functional serialization Functional
```
Comment (mcodescu) [inline OWL2/tests/test10.dol:1]: why twice here?

**Review by mcodescu — CHANGES_REQUESTED** [review]
Looks good, minor change needed.

File: OWL2/tests/test10.dol (hunk 1 above)
Comment (b-gehrke) [reply OWL2/tests/test10.dol]: Removed

**Review by mcodescu — APPROVED** [review]
:+1:

**Pull request merged**